lib/app/features/auth/providers/auth_provider.m.dart

```diff
@@ -212,3 +360,51 @@ void keepAliveWhileUnauthenticated(Ref ref) {
   final keepAlive = ref.keepAlive();
   onLogin(ref, keepAlive.close);
 }
+
+// NOTE(ice-linus): Invalidates feed, stories, and follow list providers
+// when switching between existing users to clear cached data for the previous account.
+@Riverpod(keepAlive: true)
+class PubkeyChangeWithExistingUser extends _$PubkeyChangeWithExistingUser {
```

Comment:
Why do we need to invalidate the providers manually here? Those providers should react by themselves - they are "auto-dispose" so they should be disposed automatically when the feed is disposed OR they are listening to the `ref.watch(currentPubkeySelectorProvider)` so they will be updated automatically when the user changes.